src/renderTemplate.js

```diff
@@ -0,0 +1,12 @@
+const RenderPosition = {
+  BEFOREBEGIN: 'beforebegin',
+  AFTERBEGIN: 'afterbegin',
+  BEFOREEND: 'beforeend',
+  AFTEREND: 'afterend',
+};
```

Comment:
```suggestion
const RenderPosition = {
  BEFORE_BEGIN: 'beforebegin',
  AFTER_BEGIN: 'afterbegin',
  BEFORE_END: 'beforeend',
  AFTER_END: 'afterend',
};
```

Сложные названия, принято разделять _ при именовании констант. 

ну и сделай лучше файлик consts где будут лежать общие константы и перечисления

Reply:
Исправил